app/src/main/java/org/jellyfin/androidtv/ui/playback/nextup/NextUpActivity.kt

```diff
@@ -14,13 +15,17 @@ class NextUpActivity : FragmentActivity() {
 
 	override fun onCreate(savedInstanceState: Bundle?) {
 		super.onCreate(savedInstanceState)
+		val usingExternalPlayer = intent.getBooleanExtra("usingExternalPlayer", false)
```

Comment:
We've moved extra/argument names to constants after this activity was created. Can you move the 2 extras (id and this one) to constants?

something like:
```
companion object {
    const val EXTRA_ID = "id"
}
```

Reply:
Also rename to "useExternalPlayer"